DBZ-892 Document postgres large WALs (#1082)
debezium/debezium: documentation/modules/ROOT/pages/connectors/postgresql.adoc

As mentioned link:#limitations[in the beginning], PostgreSQL (for all versions <= 12) only supports logical replication slots on `primary` servers. This means that a replica in a PostgreSQL cluster cannot be configured for logical replication, and consequently that the Debezium PostgreSQL Connector can only connect and communicate with the primary server. Should this server fail, the connector will stop. When the cluster is repaired, if the original primary server is once again promoted to `primary`, the connector can simply be restarted. However, if a different PostgreSQL server _with the plugin and proper configuration_ is promoted to `primary`, the connector configuration must be changed to point to the new `primary` server and then can be restarted.

[[wal-disk-space]]
== WAL Disk Space Consumption
In certain cases, it is possible that PostgreSQL disk space consumed by WAL file either experiences spikes or increases out of usual proportions.

> Three reasons that explain the situation

Reviewer: ```suggestion
There are three potential reasons that explain the situation:
```

This is visible as `confirmed_flush_lsn` in `pg_replication_slots` slots table.
The database is responsible for reclaiming the disk space and the WAL size can be calculated from `restart_lsn` of the same table.
So if the `confirmed_flush_lsn` is regularly increasing and `restart_lsn` lags then the database does need to reclaim the space.
Disk space is usually reclaimed in batch blocks so this is expected behaviour and no action on a user's side is necessary.
 * There are many updates in a monitored database but only minuscule amount relates to the monitored tables and/or schemas.
This situation can be easily solved by enabling periodic heartbeat events using `heartbeat.interval.ms` configuration option
 * The PostgreSQL instance contains multiple databases where one of them is a high-traffic database.
 Debezium monitors another database that is low-traffic in comparison to the other one.
 Debezium then cannot confirm the LSN as replication slot works per-database and Debezium is not invoked.
 As WAL is shared by all databases it tends to grow until an event is emitted by the database monitored by Debezium.

To overcome the third cause it is necessary to

>  * enable periodic heartbeat record generation using `heartbeat.interval.ms` configuration option

Reviewer: ```suggestion
 * enable periodic heartbeat record generation using the `heartbeat.interval.ms` configuration option
```

>  * regularly emit change events by Debezium monitored database

Reviewer: ```suggestion
 * regularly emit change events from the database tracked by Debezium
```
